src/test/java/org/springframework/integration/aws/core/RegionsTests.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.integration.aws.core;
+
+import com.amazonaws.regions.Regions;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author blake
+ * @since 2/22/17
+ */
+public class RegionsTests {
+
+    @Test
+    public void testUsEast2RegionExists() {
+        assertNotNull(Regions.US_EAST_2);
+        assertEquals("us-east-2", Regions.US_EAST_2.getName());
```

Comment:
To be honest I don't see reason in this test. 
It just won't compile if we use an older AWS SDK version.

I'd prefer that you drop it at all because we don't test here dependencies functionality, just only what we do ourselves.

And after that I'll merge it.

Thanks